week-1-practise/level-3/1-piping.js

```diff
@@ -11,7 +11,8 @@
 */
 
 var startingValue = 5;
-
+//var result = format(add(10, multiply(startingValue, 2)));
+var result = format(add(10, multiply(startingValue, 2))).match(/\d+/g);
```

Comment:
Excellent! I'm intrigued by the regular expression matcher at the end. What's that for? The solution on line 14 looks fine!

Reply:
1.  The matcher  I  used it to remove all non numeric characters from the result so the out put will be just the Expected result: 20,00  without the pound symbol from the the function format 
2. The solution on line 14 is fine put the result will be £ 20.00 not just 20.00 
3. I was trying to find a matcher that will return  20.00 put i could not . so if you know a mathcer that will return 20.00 I will be more than happy to use .

Reply:
I see! I think maybe the expected result is wrong in this case 🤔